externs/browser/w3c_geolocation.js

```diff
@@ -78,14 +78,14 @@ GeolocationPosition.prototype.coords;
 
 
 /**
- * @constructor
+ * @record
  * @see http://www.w3.org/TR/geolocation-API/#position-options
  */
 function GeolocationPositionOptions() {}
-/** @type {boolean} */
+/** @type {boolean|undefined} */
 GeolocationPositionOptions.prototype.enableHighAccuracy;
-/** @type {number} */ GeolocationPositionOptions.prototype.maximumAge;
-/** @type {number} */ GeolocationPositionOptions.prototype.timeout;
+/** @type {number|undefined} */ GeolocationPositionOptions.prototype.maximumAge;
```

Comment:
nit: Can you add a newline after the jsdoc comment like we have for enableHighAccuracy? Makes it easier to quickly scan through the file visually and see all the properties.